sdks/python/apache_beam/examples/snippets/transforms/element_wise/pardo_test.py

```diff
@@ -61,16 +59,23 @@ def __init__(self, methodName):
 window.start -> Timestamp(1584675660) (2020-03-20 03:41:00)
 window.end -> Timestamp(1584675690) (2020-03-20 03:41:30)
 window.max_timestamp() -> Timestamp(1584675689.999999) (2020-03-20 03:41:29.999999)'''
-    # [END dofn_params]
-    # pylint: enable=line-too-long
-    self.dofn_params_test = lambda actual: \
-        assert_that(actual, equal_to([dofn_params]))
+  # [END dofn_params]
+  # pylint: enable=line-too-long
+  assert_that(actual, equal_to([dofn_params]))
 
+
+@mock.patch('apache_beam.Pipeline', TestPipeline)
+# pylint: disable=line-too-long
+@mock.patch('apache_beam.examples.snippets.transforms.element_wise.pardo.print', lambda elem: elem)
+# pylint: enable=line-too-long
+class ParDoTest(unittest.TestCase):
   def test_pardo_dofn(self):
-    pardo_dofn(self.plants_test)
+    pardo_dofn(check_plants)
 
+  @unittest.skipIf(sys.version_info[0] < 3 and platform.system() == 'Windows',
+                   'Python 2 on Windows uses`long` rather than `int`')
```

Comment:
nit: missing space between uses and `long`

Reply:
Good eye, fixed